lib/tasks/jenkins.rake

```diff
@@ -0,0 +1,23 @@
+begin
+  require 'English'
+rescue LoadError
+else
+  namespace :jenkins do
+    task :unit => ['jenkins:setup:minitest', 'rake:test']
+
+    namespace :setup do
+      task :minitest do
+        ENV['MINITEST_REPORTER'] = 'JUnitReporter'
+        ENV['MINITEST_REPORTERS_REPORTS_DIR'] = 'jenkins/reports/unit'
+      end
+    end
+
+    task :rubocop do
+      system("bundle exec rubocop \
```

Comment:
I think this isn't really needed anymore and we should migrate away from it, in favor of `recordIssues` with the `ruboCop` parser. See https://github.com/theforeman/jenkins-jobs/blob/1041458b9bb8e68e53b8d863e0ff251540632304/theforeman.org/pipelines/test/smart-proxy.groovy#L45-L54

Again, a lot of legacy we don't really need anymore, but haven't gotten around to cleaning up.